x/subkeys/internal/types/tx/tx.go

```diff
@@ -0,0 +1,241 @@
+package tx
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/tendermint/tendermint/crypto"
+	"github.com/tendermint/tendermint/crypto/multisig"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/exported"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+)
+
+var (
+	_ sdk.Tx = DelegatedTx{}
+
+	maxGasWanted = uint64((1 << 63) - 1)
+)
+
+// DelegatedTx is wraps a Msg with Fee and Signatures,
+// adding the ability to delegate the fee payment
+// NOTE: the first signature responsible for paying fees, either directly,
+// or must be authorized to spend from the provided Fee.FeeAccount
+type DelegatedTx struct {
```

Comment:
DelegatedTx sounds like a staking type transaction, any chance we can rename it?

Reply:
Yes, please. I need some consensus on naming. I suck at it.

Just decide what names to use here, and I will use them. (Ideally the discussion off of the PR and the final decision on the PR)